jupyter-extension/src/plugins/mount/customFileBrowser.ts

```diff
@@ -43,7 +43,7 @@ const createCustomFileBrowser = (
     // select and scroll to the file opened on a delay. The file attempting to be selected may not be visible and
     // if it is then it interferes with the user scrolling immediately.
     state: null,
-    auto: true,
+    auto: false,
```

Comment:
Maybe add a comment here explaining this issue and why it's important auto: false is set? I am sort of on the fence on if you want to setup a cypress test to make sure the popups don't appear with no config file since as you mentioned it could be pretty tricky. As long as we have a comment here explaining it's important and test the other two parts that seems fine.